spec/imsc-hrm.html

```diff
@@ -532,7 +535,7 @@ <h2>Paint Images</h2>
     <section id='paint-text'>
       <h2>Paint Text</h2>
 
-      <p>In the context of this section, a <dfn>glyph</dfn> is a tuple consisting of (i) one character and (ii) the computed values
+      <p>In the context of this section, a <dfn>glyph</dfn> is a tuple consisting of (i) one [[UNICODE]] code point and (ii) the computed values
```

Comment:
_character_ is define in [TTML2](https://www.w3.org/TR/ttml2/#reduced-infoset-character) as far as I know. I do not think we need to redefine it here.

Reply:
The term in TTML2 is "character information item" and references back to [XML Infoset's definition](https://www.w3.org/TR/2004/REC-xml-infoset-20040204/#infoitem.character), which ends up, for practical purposes, meaning nearly the same as the text here, but via a more tortuous route. 

The problem I see with Character Information Item as defined is that it formally includes properties that we certainly do not want to include here, like the parent element information item. Even if two code points with the same Unicode numerical value belong to different parent elements, for the purposes of the HRM, we need them to be considered identical, so we really do not want that property to be included.

I think the proposal I have is clearer and says exactly what we want. I'd be happy to switch the Unicode reference to ISO 10646 to coincide with the XML Infoset definition of character information item, but I don't think that would make any practical difference.

Reply:
@nigelmegitt The issue in my mind is that XML parsers read "characters" out of the IMSC document, so somewhere there has to be a connection between whatever term is used in the definition of _HRM glyph_ and XML character.

Reply:
OK I will reinstate "character" and define it as the character code property of a character information item as defined by TTML2.